docs/roadmap.md

```diff
@@ -18,9 +18,18 @@ The goal of this doc is to align *core* and *community* efforts for the project
 
 ## What is the core Chroma team working on right now?
 
+This week we are working through the backlog of Github issues and PRs and adding lots of new functionality. A few issues we are looking to address:
```

Comment:
Do we plan on updating every week?

Reply:
I don't think that's unreasonable! We will see :)